### PR TITLE
docs: Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by patricebender**
Add all types which are acceptable for our commits according to #281

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -53,8 +53,18 @@ The `type` can be any of `feat`, `fix` or `chore`.
...
 
-| **type**  | when to use                         | release level | release note section |
-| --------- | ----------------------------------- | ------------- | -------------------- |
-| feat      | a feature has been added            | `minor`       | **Features**         |
-| fix       | a bug has been patched              | `patch`       | **Bug fixes**        |
-| chore     | any changes that aren't user-facing | none          | none                 |
+| **type**   | When to Use                          | Release Level | Release Note Section  |
+| ---------- | ----------------------------------- | ------------- | --------------------   |
+| feat       | A feature has been added            | `minor`       | **Features**           |
+| fix        | A bug has been patched              | `patch`       | **Bug fixes**          |
```
Comment (patricebender) [inline CONTRIBUTING.md:59]:
```suggestion
| fix        | A bug has been patched              | `patch`       | **Bug fixes**          |
| deps        | Changes to the dependencies          | `patch`       | **Changed**          |
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -53,8 +53,18 @@ The `type` can be any of `feat`, `fix` or `chore`.
...
+| chore      | Any changes that aren't user-facing | none          | none                   |
+| docs       | Documentation updates               | none          | none                   |
+| style      | Code style and formatting changes   | none          | none                   |
+| refactor   | Code refactoring                    | none          | none                   |
+| perf       | Performance improvements            | none          | none                   |
```
Comment (patricebender) [inline CONTRIBUTING.md:64]:
```suggestion
| perf       | Performance improvements            | none          | **Performance Improvements**                   |
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -53,8 +53,18 @@ The `type` can be any of `feat`, `fix` or `chore`.
...
 The prefix is used to calculate the semver release level, and the section of the release notes to place the commit message in.
 
-| **type**  | when to use                         | release level | release note section |
-| --------- | ----------------------------------- | ------------- | -------------------- |
-| feat      | a feature has been added            | `minor`       | **Features**         |
-| fix       | a bug has been patched              | `patch`       | **Bug fixes**        |
-| chore     | any changes that aren't user-facing | none          | none                 |
+| **type**   | When to Use                          | Release Level | Release Note Section  |
+| ---------- | ----------------------------------- | ------------- | --------------------   |
+| feat       | A feature has been added            | `minor`       | **Features**           |
```
Comment (johannes-vogel) [inline CONTRIBUTING.md:58]:
```suggestion
| feat       | A feature has been added            | `minor`       | **Added**           |
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (johannes-vogel) [inline CONTRIBUTING.md:59]:
```suggestion
| fix        | A bug has been patched              | `patch`       | **Fixed**          |
```

**Comment by patricebender** [comment]
@johannes-vogel good to go?

**Pull request merged**